doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -257,3 +296,21 @@ Adjust frame(s) of selected key(s)              :kbd:`Shift` + click and drag le
 Adjust value(s) of selected key(s)              :kbd:`Shift` + click and drag up or down
 =============================================== =============================================
 ```
+
+
+## Plug Values ##
+
+> Note :
+> The following controls and shortcuts only operate on numerical plugs.
+
+```eval_rst
+====================================================== ==============================================
+Action                                                 Control or Shorcut
+====================================================== ==============================================
+Increment/decrement plug value, arbitrary precision    Edit plug field, position cursor next to
```

Comment:
Not sure if "arbitrary" is apt. "Exact"? "Specific"?

Reply:
"Specific" sounds good to me.